src/client/cosmos.ts

```diff
@@ -3,20 +3,21 @@
  * Licensed under the MIT License.
  */
 
-import * as url from 'url';
-
 import {
   ConnectionPolicy,
   Container,
   ContainerDefinition,
   CosmosClient,
   CosmosHeaders,
   FeedResponse,
+  ItemDefinition,
   Resource,
   ResourceResponse,
   Response,
   SqlQuerySpec
 } from '@azure/cosmos';
+import { ChainedTokenCredential } from '@azure/identity';
```

Comment:
it's usually bad practice to re-export types/objects from an external package like this. i would expect this to be a peer dependency instead so people can install whatever version makes sense for them